docs/spec/lsm/adr-001.md

```diff
@@ -0,0 +1,104 @@
+# ADR-001: Exempt delegations as a transitionary mechanism for safe adoption of liquid staking.
```

Comment:
Why are there sub-ADRs? Should these not be moved to the parent ADR section?

Reply:
Resolved, they're in ADR61 now.